### ISSUE TITLE
Incompatible with show: "Foundation (2021)"

### ISSUE THREAD

**Issue by codedesperate** [Shraymonks/unmonitorr#9]
Hi Shraymonks, 
I've found that there is one specific show "unmonitorr" cannot unmonitor. It is "Foundation (2021)". I'm thinking myabe the "()" is throwing it off?

In the logs I see the following: (And as you can see, it can unmonitor everything else just fine, both movies like barbie, and tv shows like ahsoka)

```
root@CENSORED:/home/CENSORED# docker logs unmonitorr3
unmonitoring for media.play by account(CENSORED) on port: 9799
Radarr: http://127.0.0.1:7883
Sonarr: http://127.0.0.1:8993
Barbie (2023) unmonitored!
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Ahsoka - S1E4 unmonitored!
unmonitoring for media.play by account(CENSORED) on port: 9799
Radarr: http://127.0.0.1:7883
Sonarr: http://127.0.0.1:8993
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Barbie (2023) unmonitored!
The Ritual Killer (2023) unmonitored!
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Ahsoka - S1E5 unmonitored!
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
Could not find Foundation (2021) in sonarr library
root@CENSORED:/home/CENSORED
```

**Comment by Shraymonks** [comment]
Looks like this can happen for some shows with the same title where Plex includes the year as part of the title in its metadata. I can make some changes in the series matching to account for this.

**Comment by codedesperate** [comment]
That would be great! :D

**Comment by Shraymonks** [comment]
Should be fixed now in [unmonitorr@0.3.0](https://github.com/Shraymonks/unmonitorr/pull/10)

**Comment by codedesperate** [comment]
Just updated and tested, works like a charm!
